doc/ref/gappkg.xml

```diff
@@ -1286,8 +1286,8 @@ This means that the binary is not regarded as essential for this
 package.
 <P/>
 You might also have to cope with the situation that external binaries will
-only run under UNIX (and not, say, under Windows), or may not compile with
-some compilers or default compiler options.
+only run under UNIX (and not e.g. under Windows), or may not compile
+with some compilers or default compiler options.
```

Comment:
Optional and unimportant, but: could move "with" back to the previous line, to reduce the diff